### DISCUSSION TITLE
Is there a program that can check m3u8 servers in the channel list?

### DISCUSSION THREAD

**Discussion by peterfighter777** [iptv-org/community#591] [Q&A]
For example, there are over 1,000 US channels, but when I click on them, there are too many that say the server is down or cannot be played.

It's too hard to check and delete them one by one,

so I wonder if there is a program that automatically deletes downed servers from the list.

And it would be nice to have a simple list that only collects general US channels, such as ABC, PBS, CW, TBS, etc.

**Comment by ghost** [comment]
> so I wonder if there is a program that automatically deletes downed servers from the list.

some channels are not 24/7, some geo/blocked. I think it will be difficult to maintain. Although it is possible to do it the way IPTVCAT does, like we run all channels that are not marked as not 24/7 and geoblocked, and if they have not been online for a week or two, then delete them. It would be possible to write a github workflow, but it seems to me that it will take a very long time to complete and will exceed the limit of 2000 minutes per month. maybe someone has an extra computer/server to run a self-hosted workflow
